crates/vm/levm/src/opcode_handlers/environment.rs

```diff
@@ -467,29 +456,20 @@ impl VM {
             address,
         );
 
-        // https://eips.ethereum.org/EIPS/eip-7702#delegation-designation
-        let is_delegation = has_delegation(&account_info)?;
-
         current_call_frame.increase_consumed_gas(gas_cost::extcodehash(
             address_was_cold,
             self.env.config.fork,
         )?)?;
 
-        if is_delegation {
-            let hash =
-                U256::from_big_endian(keccak(&SET_CODE_DELEGATION_BYTES[..2]).as_fixed_bytes());
-            current_call_frame.stack.push(hash)?;
-        } else {
-            // An account is considered empty when it has no code and zero nonce and zero balance. [EIP-161]
-            if account_info.is_empty() {
-                current_call_frame.stack.push(U256::zero())?;
-                return Ok(OpcodeResult::Continue { pc_increment: 1 });
-            }
-
-            let hash = U256::from_big_endian(keccak(account_info.bytecode).as_fixed_bytes());
-            current_call_frame.stack.push(hash)?;
+        // An account is considered empty when it has no code and zero nonce and zero balance. [EIP-161]
+        if account_info.is_empty() {
+            current_call_frame.stack.push(U256::zero())?;
+            return Ok(OpcodeResult::Continue { pc_increment: 1 });
         }
 
+        let hash = U256::from_big_endian(keccak(account_info.bytecode).as_fixed_bytes());
```

Comment:
Let's explicitly make the keccack use to_le_bytes instead

Reply:
There isn't a `to_be_bytes` present, we only have `as_fixed_bytes`. There may be a "clearer" way for this:
```suggestion
        let hash = U256::from_str(&keccak(account_info.bytecode).to_string())
            .map_err(|_| VMError::Internal(InternalError::ConversionError))?;
```
In this case, the `from_str` and `to_str` handle the byte ordering internally without exposing it to the user (us).

Reply:
If you think it's a better approach feel free to commit it.

Reply:
> Let's explicitly make the keccack use to_le_bytes instead

I'm pretty sure this is the best way to turn an `H256` into an `U256` (sadly).